app/src/main/java/com/github/wnder/GuessPreviewActivity.java

```diff
@@ -15,13 +18,23 @@
     public static final String EXTRA_CAMERALAT = "cameraLat";
     public static final String EXTRA_CAMERALNG = "cameraLng";
 
+    private Button guessButton;
+    private Button skipButton;
+    private Button reportButton;
+
```

Comment:
Is it necessary to keep it as a variable if the button in itself will not change (except maybe the listener associated)?

Reply:
It is true that for now those private variables are useless.
I removed them.